http/src/test/groovy/io/micronaut/http/uri/UriBuilderSpec.groovy

```diff
@@ -23,7 +23,7 @@ class UriBuilderSpec extends Specification {
 
     void "test uri builder expand"() {
         given:
-        def builder = UriBuilder.of("/person/{name}")
+        def builder = UriBuilder.of("/person/{name}", UrlEncodingKind.RFC_3986)
```

Comment:
Can you copy the test with the new option instead of modifying existing? Or does It change all the default behaviour